custom_components/tahoma/climate_devices/stateless_exterior_heating.py

```diff
@@ -0,0 +1,93 @@
+"""Support for Atlantic Electrical Heater IO controller."""
+import logging
+from typing import List, Optional
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    HVAC_MODE_HEAT,
+    HVAC_MODE_OFF,
+    PRESET_BOOST,
+    PRESET_COMFORT,
+    PRESET_ECO,
+    SUPPORT_PRESET_MODE,
+)
+from homeassistant.const import TEMP_CELSIUS
+
+from ..tahoma_device import TahomaDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+COMMAND_DOWN = "down"
+COMMAND_MY = "my"
+COMMAND_OFF = "off"
+COMMAND_ON = "on"
+COMMAND_UP = "up"
+
+SUPPORT_MY = pow(2, 20)
+
+
+class StatelessExteriorHeating(TahomaDevice, ClimateEntity):
+    """Representation of TaHoma Stateless Exterior Heating device."""
+
+    def __init__(self, tahoma_device, controller):
```

Comment:
For debug purposes I guess, can be removed afterwards. What you pass here by the way `device_url` followed by `coordinator`, not the device and controller.

Reply:
will be removed once working!